lib/cli.js

```diff
@@ -34,8 +36,13 @@ files.forEach(function (file) {
 
     var json5 = FS.readFileSync(path, 'utf8');
     var obj = JSON5.parse(json5);
-    var json = JSON.stringify(obj, null, 4); // 4 spaces; TODO configurable?
-
-    path = Path.join(dirname, basename + '.json');
-    FS.writeFileSync(path, json, 'utf8');
+    // if we are here for validation, we'll just output things are OK.
+    if(args[2] === '--validate') {
+      console.log('\t\x1b[32;1m' + file + ' is valid.\x1b[0m');
```

Comment:
What does the output look like for an invalid file? I assume `JSON5.parse` will throw an error, so you'll just get the stack trace. But that doesn't seem any different from what you reported in <https://github.com/aseemk/json5/issues/72#issuecomment-59616865>?

The fix, I *think*, would be to wrap the whole `FS.readFileSync` + `JSON5.parse` in a try/catch, and in the catch, log "<file> is invalid" along w/ the error `stack` trace. (But important: re-throw the error if compiling.)